tests/test.rs

```diff
@@ -1,7 +1,7 @@
 #[macro_use]
 mod compiletest;
 
-#[rustversion::attr(not(nightly), ignore)]
+#[rustversion::attr(not(since(1.67.0)), ignore)]
```

Comment:
I suspect we should be able to safely lower the minimum Rust version for ui tests. Most important for me is to enable the ui tests without having to use the nightly version. Thanks in advance for your consideration.